README.md

```diff
@@ -112,7 +112,13 @@ Its work is hosted and supported by the [Node.js Foundation](https://nodejs.org/
 
 ## React
 
-[React](https://facebook.github.io/react/) section to be completed.
+[React](https://facebook.github.io/react/) is a library developed and used at Facebook for building user interfaces. It can be seen as the V in MVC as it makes no assumptions about the rest of the technology stack. Using React Native it can even be used to power native apps.
+
+In React you can write HTML directly in JS using an XML-like syntax called JSX. JSX compiles to JS and is optional, but does make the code more expressive.
+
+Data flow in React is one-way which makes it easier to reason about and avoid mistakes. This quality can be enhanced using Flux, Facebook’s complementary application architecture, or Redux which many people see as a “better Flux”.
+
+React is not a framework.
```

Comment:
Kind of weird to finish with is. I would have actually said it is a framework. :x